### ISSUE TITLE
Make sure the content matches the objectives for each episode

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#288]
Necessary to meet the requirements for carpentries lab. See #264

**Comment by dsmits** [comment]
Episode 1:
1. [x] Recall the sort of problems for which Deep Learning is a useful tool
2. [x] List some of the available tools for Deep Learning
3. [x] Recall the steps of a Deep Learning workflow
4. [ ] Explain why it is important to test the accuracy of Deep Learning system.
5. [ ] Identify the inputs and outputs of a Deep Learning system.
6. [x] Test that we’ve correctly installed the Keras, Seaborn and Sklearn libraries

For this one, I'd say we ditch number 4 and rephrase the 5th one to say "identify the components of a single neuron" or something like that

**Comment by dsmits** [comment]
Episode 2:
1. [x] Use the deep learning workflow to structure the notebook
2. [x] Explore the dataset using pandas and seaborn
3. [x] Use one-hot encoding to prepare data for classification in Keras
4. [x] Describe a fully connected layer
5. [x] Implement a fully connected layer with Keras
6. [x] Use Keras to train a small fully connected network on prepared data
7. [x] Plot the loss curve of the training process
8. [x] Use a confusion matrix to measure the trained networks’ performance on a test set

**Comment by dsmits** [comment]
Episode 4:
1. [x] Understand why convolutional and pooling layers are useful for image data
2. [ ] Use normalization as preparation step for Deep Learning
3. [x] Implement a convolutional neural network on an image dataset
4. [x] Plot and interpret the training process
5. [ ] Do a visual inspection of the results of your neural network
6. [ ] Understand strategies to improve your model based on the plots
7. [x] Use drop-out layer to prevent overfitting

- Number 2: should be moved to episode 3
- Number 5 and 6 are kind of there but are more an objective of the previous chapter
